package.json

```diff
@@ -1,55 +1,18 @@
 {
-  "name": "monolint",
-  "version": "0.0.5",
-  "description": "Monorepo linter for checking folder structure and file contents",
-  "main": "dist/main.js",
-  "bin": "dist/main.js",
-  "files": [
-    "dist/main.js"
-  ],
+  "name": "@monolint/root",
   "scripts": {
```

Comment:
Why do we have this script in the root package.json?

Reply:
Removed :)